.changeset/stupid-rings-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@web3-ui/components': minor
+---
+
+Add TokenInput component
```

Comment:
Since this is added to the changelog, it'd be great to add more detail so our users understand _what_ this component is, _why_ it's useful, and _how_ they can use it.

````suggestion
Added the `TokenInput` component which combines an `input` for the user to provide an amount of tokens and a `select` for the user to select the type of token. 

```tsx
const Example = () => {
  const [value, setValue] = React.useState('');
  const [token, setToken] = React.useState('');
  
  return (
    <TokenInput
    	tokens={['BTC', 'ETH', 'GTC']}
      value={value}
      onValueChange={event => setValue(event.target.value)}
      selectedToken={token}
      onTokenChange={event => setToken(event.target.value)}
    />
  )
}
```
````

Reply:
Agreed, I'll add more info in the upcoming commits